src/pocketmine/lang/Installer/en.ini

```diff
@@ -26,20 +26,20 @@ op_who = OP player name (example, your game name)
 op_warning = You will be able to add an OP user later using /op <player>
 whitelist_info = The white-list only allows players in it to join.
 whitelist_enable = Do you want to enable the white-list?
-whitelist_warning = You will have to add the players to the white-list
+whitelist_warning = You will have to add the players to the white-list.
 
 query_warning1 = Query is a protocol used by different tools to get information of your server and players logged in.
 query_warning2 = If you disable it, you won't be able to use server lists.
 query_disable = Do you want to disable Query?
 rcon_info = RCON is a protocol to remote connect with the server console using a password.
 rcon_enable = Do you want to enable RCON?
 rcon_password = RCON password (you can change it later)
-usage_info = The anonymous usage data allows us to calculate global statistics for PocketMine-MP and its plugins. You can view them on stats.pocketmine.net
+usage_info = The anonymous usage data allows us to calculate global statistics for PocketMine-MP and it's plugins. You can view them on stats.pocketmine.net
```

Comment:
It was correct. It was a possessive pronoun for "PocketMine-MP's plugins", not "it is plugins".